nessie/src/main/java/org/apache/iceberg/nessie/NessieIcebergClient.java

```diff
@@ -85,10 +85,14 @@ public NessieApiV1 getApi() {
     return api;
   }
 
-  public UpdateableReference getRef() {
+  UpdateableReference getRef() {
```

Comment:
`UpdateableReference` is a package private class. So, returning it from a public method is no use for callers outside the package.